src/store/getHandlers.js

```diff
@@ -0,0 +1,7 @@
+let { extend } = require('../mindash');
+
+function getHandlers(store) {
+  return extend({}, store.handlers, store.constructor.handlers);
```

Comment:
Cool - no need to muck with `hasOwnProperty` or whatever in this case like for `Application#register`!